packages/babel-node/src/_babel-node.ts

```diff
@@ -151,8 +143,8 @@ const _eval = function (code: string, filename: string) {
 
   code = babel.transformSync(code, {
     filename: filename,
-    presets: opts.presets,
-    plugins: (opts.plugins || []).concat([replPlugin]),
+    ...babelOptions,
+    plugins: (opts.babelOptions || []).concat([replPlugin]),
```

Comment:
```suggestion
    plugins: (opts.plugins || []).concat([replPlugin]),
```
`babelOptions` is not a valid option name anyway.

Reply:
Oh sorry, my mistake, just wanted to use babelOptions.plugins here